core/src/test/scala/unit/kafka/tools/StorageToolTest.scala

```diff
@@ -54,7 +54,7 @@ class StorageToolTest {
     properties
   }
 
-  val allFeatures = Features.FEATURES.toList
+  val testingFeatures = Features.values().toList.asJava
```

Comment:
Why do we need to change this?

Reply:
It was unused before and not the right type. I can also use FEATURES.toList.asJava if that is preferred.

Reply:
I guess the intention of Features.FEATURES is to hide the usage of Features.values()?